packages/boxel/addon/components/boxel/dropdown/index.gts

```diff
@@ -19,17 +20,37 @@ interface Signature {
 }
 
 // Needs to be class, BasicDropdown doesn't work with const
-class BoxelDropdown extends Component<Signature> { 
+class BoxelDropdown extends Component<Signature> {
   <template>
     <BasicDropdown as |dd|>
-      <dd.Trigger ...attributes>
+      <dd.Trigger
```

Comment:
What If we add a `display: inline-block;` on `ember-basic-dropdown-trigger` selector ? it would at least wrap the triggers content, to avoid the selection outside of the trigger button.

<img width="336" alt="image" src="https://user-images.githubusercontent.com/20520102/192798858-d3ed9513-0831-42c3-a3a9-3d86c29f1a31.png">

Now regarding the interactives, maybe if we pass use `triggerComponent` param ? and create a custom trigger that handles other events to avoid bubbling up, not sure it it's possible but we can think about later.

Reply:
Yeah, I think `@triggerComponent` seems like the way that `ember-basic-dropdown` would want us to solve this problem. If we do that we won't have to bother about trigger vs contained interactive element, since the `@triggerComponent` itself is the trigger. It is kind of awkward passing components in this way, but I think with template imports and the ability to create a component within the same file, it might be ok?

Reply:
The only awkward thing is that I don't think you can attach a modifier to {{component ....}} helper call. :\